tests/validation/check_validation_test.go

```diff
@@ -0,0 +1,132 @@
+package validation
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/cenkalti/backoff/v4"
+	parser "github.com/craigpastro/openfga-dsl-parser/v2"
+	"github.com/openfga/openfga/cmd"
+	"github.com/openfga/openfga/pkg/testfixtures/storage"
+	"github.com/openfga/openfga/pkg/typesystem"
+	"github.com/stretchr/testify/require"
+	pb "go.buf.build/openfga/go/openfga/api/openfga/v1"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	healthv1pb "google.golang.org/grpc/health/grpc_health_v1"
+	"google.golang.org/grpc/status"
+	"gopkg.in/yaml.v3"
+)
+
+type checkValidationTests struct {
+	Tests []checkValidationTest
+}
+
+type checkValidationTest struct {
+	Name    string
+	Model   string
+	Tuples  []*pb.TupleKey
```

Comment:
We don't provide any tuples in the tests since these are all request validation related errors. We can omit, yeah?

Reply:
Good point. We don't yet so I'll remove it. Thanks!